js/index.js

```diff
@@ -1 +1,69 @@
-// JS goes here
\ No newline at end of file
+// JS goes here
+
+// ************************ tab navigator *************************
+class Tab {
+    constructor(tab) {
+        this.tab = tab;
+        this.tabMatcher = this.tab.dataset.matcher;
+        this.tabCard = document.querySelector(`.card[data-matcher="${this.tabMatcher}"]`);
+        this.tabCard = new Card(this.tabCard);
+        this.tab.addEventListener('click', (event) => {this.tabClick(event)});
+    }
+
+    tabClick() {
+        this.tabCard.toggleContent();
+        let tabs = document.querySelectorAll('.tab');
+        tabs.forEach(tab => tab.classList.remove('tabSelected'));
+        this.tab.classList.toggle('tabSelected');
+    }
+}
+
+class Card {
+    constructor(tabCard) {
+        this.tabCard = tabCard;
+    }
+
+    toggleContent() {
+        let cards = document.querySelectorAll('.card');
+        cards.forEach(card => card.classList.remove('showCard'));
+        this.tabCard.classList.toggle('showCard');
+    }
+}
+
+// only run this code on Services.html in order to avoid error when '.tab' can't be found
+let activePg = document.querySelector('title').innerText;
```

Comment:
may make sense to split your javascript into two separate files, so you're not loading scripts that don't apply to a given page